### PR TITLE
Add Stop task event

### PR DESCRIPTION
**Pull request by mushixun**
<<Replace this line with your Revision Title>>

Summary:
For more details follow http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html.

JIRA issues:
https://jira.mesosphere.com/browse/MARATHON-8339

### CODE REVIEW

**Comment by mushixun** [comment]
How to get mesphere onglogin,I  can not view ci log。

**Comment by alenkacz** [comment]
@mushixun Hello! Thank you for the PR.

Can you please talk a bit more about your intentions in comments here https://jira.mesosphere.com/browse/MARATHON-8339

Before accepting the PR we need to understand why InstanceChange event is not an option for you. You should get one shortly after the kill was initiated.

**Comment by mushixun** [comment]
@alenkacz OK,When using marathon-lb for load balancing, when marathon is shrinking, some requests fail. After analysis, it is because the container has not been removed from the haproxy routing table when the container is killed. It is not caused by haproxy reload, so I hope marathon can tell which container to shrink in advance when shrinking. So that I can first remove the container information from haproxy's routing table, no longer accept new requests, and then kill the container, and the premise of this function is that marathon needs to issue the first stop when shrinking. send stop event information.

**Comment by mushixun** [comment]
Tell marathon-lb that which containers will be stop

File: .gitignore [hunk 1]
```diff
@@ -1,29 +0,0 @@
-bin/run-tests-config.sh
```
Comment (ichernetsky) [inline .gitignore]: Please restore this file.

**Comment by ichernetsky** [comment]
```
src/test/scala/mesosphere/marathon/core/task/termination/impl/KillServiceActorTest.scala:310:54: not enough arguments for method props: (driverHolder: mesosphere.marathon.MarathonSchedulerDriverHolder, instanceTracker: mesosphere.marathon.core.task.tracker.InstanceTracker, config: mesosphere.marathon.core.task.termination.KillConfig, clock: java.time.Clock, eventStream: akka.event.EventStream)akka.actor.Props.
[error] Unspecified value parameter eventStream.
[error]     val actor = system.actorOf(KillServiceActor.props(f.driverHolder, f.instanceTracker, killConfig, f.clock), s"KillService-${UUID.randomUUID()}")
```

**Pull request closed without merging**